icons/pilcrow-right.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11 8.56H8.22a2.78 2.78 0 0 1 0-5.56h7.89M11 3v11M15 3v11M21 18H3M18 15l3 3-3 3" />
```

Comment:
I feel like with the left shift, it's fine.
But shifting pilcrow-left to the left makes it feel quite left heavy.
```suggestion
  <path d="M10 8.5H7A1 1 0 0 1 7 3h8" />
  <path d="M10 3v11" />
  <path d="M14 3v11" />
  <path d="M21 18H3" />
  <path d="m18 15 3 3-3 3" />
```

Reply:
Nice work splitting up those paths, @jguddas
Also see https://github.com/lucide-icons/lucide/pull/1667#issuecomment-1804665782

Unfortunately the pilcrow itself is off grid and hence appears pretty blurry:
![image](https://github.com/lucide-icons/lucide/assets/17746067/6039c2e0-142e-4e8f-88f0-15b7bbb5ff66)

Reply:
My suggestion:
- split the paths
- align it with the grid
- move it a bit to the left

Result:
![](https://lucide.dev/api/gh-icon/dpi/24/ICA8cGF0aCBkPSJNMTAgOUg3YTEgMSAwIDAgMSAwLTZoOCIgLz4KICA8cGF0aCBkPSJNMTAgM3YxMSIgLz4KICA8cGF0aCBkPSJNMTQgM3YxMSIgLz4KICA8cGF0aCBkPSJNMjEgMThIMyIgLz4KICA8cGF0aCBkPSJtMTggMTUgMyAzLTMgMyIgLz4=.svg)

Reply:
@zaaakher Can you apply those changes from @jguddas?